### PR TITLE
add the availability time to the project page

### PR DESCRIPTION
**Pull request by JFleming4**
and found/fixed a bug when updating availability to none

### CODE REVIEW

File: src/main/resources/templates/project.html [hunk 1]
```diff
@@ -35,6 +35,10 @@ <h3>Students:</h3>
     <div>
         <hr/>
         <h3>Oral Presentation</h3>
+        <div th:style="${selectedTime != null} ? 'display:block' : 'display:none'">
```
Comment (DerekStride) [inline src/main/resources/templates/project.html]: use `th:unless="${selectedTime == null}"`

File: src/main/resources/templates/presentation.html [hunk 2]
```diff
@@ -27,6 +27,7 @@
                                 <span th:if="${timeslot.getStartMinute() == 0}" th:text="${timeslot.getDay()}+' '+${timeslot.getStartHour()}+':'+${timeslot.getStartMinute()}+'0'" />
                                 <span th:unless="${timeslot.getStartMinute() == 0}" th:text="${timeslot.getDay()}+' '+${timeslot.getStartHour()}+':'+${timeslot.getStartMinute()}+''" />
                             </li>
+                            <li ></li>
```
Comment (DerekStride) [inline src/main/resources/templates/presentation.html]: ?

File: src/main/resources/templates/project.html (hunk 1 above)
Comment (JFleming4) [reply src/main/resources/templates/project.html]: that doesn't work because i need to actively assign the style

Comment (JFleming4) [reply src/main/resources/templates/project.html]: I'd need an if else statement

Comment (JFleming4) [reply src/main/resources/templates/project.html]: i was wrong
i fixed it

**Review by DerekStride — APPROVED** [review]

**Pull request merged**